K8s-deployment/Charts/consent-validator/values.yaml

```diff
@@ -0,0 +1,584 @@
+
+## @section Image Parameters
+
+## %%MAIN_CONTAINER/POD_DESCRIPTION%%
+##
+
+  ## Bitnami %%MAIN_CONTAINER%% image
+  ## ref: https://hub.docker.com/r/bitnami/%%IMAGE_NAME%%/tags/
+  ## @param image.registry %%MAIN_CONTAINER%% image registry
+  ## @param image.repository %%MAIN_CONTAINER%% image repository
+  ## @param image.tag %%MAIN_CONTAINER%% image tag (immutable tags are recommended)
+  ## @param image.pullPolicy %%MAIN_CONTAINER%% image pull policy
+  ## @param image.pullSecrets %%MAIN_CONTAINER%% image pull secrets
+  ## @param image.debug Enable %%MAIN_CONTAINER%% image debug mode
+  ##ghcr.io/datakaveri/consent-validator:1.0.0-alpha-4b0a2bf
+image:
+  registry: ghcr.io
+  repository: datakaveri/consent-validator
+  tag: 1.0.0-alpha-4b0a2bf
+  pullSecrets: 
+    ## Specify a imagePullPolicy
+    ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+    ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+    ##
+  pullPolicy: IfNotPresent
+    ## Optionally specify an array of imagePullSecrets.
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ## e.g:
+    ## pullSecrets:
+    ##   - myRegistryKeySecretName
+    ##
+    ## Enable debug mode
+    ##
+  debug: false
+  
+  ## @param containerPorts.http HTTP container port
+  ## @param containerPorts.https  HTTPS container port
+  ## @param containerPorts.hazelcast  Hazelcast container port
+  ## @param containerPorts.prometheus  Prometheus container port
+
+containerPorts:
+  http: 8080
+  #https: 443
+  hazelcast: 5701
+  prometheus: 9000
+
+## @param podAnnotations Annotations for pods
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+
+podAnnotations:
+  prometheus.io/port: "9000"
+  prometheus.io/scrape: "true"
+
+## @section consentValidator Parameters
+consentValidator:
+## @param consentValidator.replicaCount Number of consentValidator replicas to deploy
+  ##
+  replicaCount: 1
+  
+  ## Configure extra options for consentValidator containers' liveness and readiness probes
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param consentValidator.livenessProbe.enabled Enable livenessProbe on consentValidator containers
+  ## @param consentValidator.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param consentValidator.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param consentValidator.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param consentValidator.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param consentValidator.livenessProbe.successThreshold Success threshold for livenessProbe
+  ## @param consentValidator.livenessProbe.path Path for httpGet
+  ##
+  livenessProbe:
+    enabled: false
```

Comment:
Why is it disabled?

Reply:
currently it does not have metrics

Reply:
metrics will be added, we can wait till then